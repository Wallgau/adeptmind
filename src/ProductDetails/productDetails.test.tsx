import React from 'react'
import renderer from 'react-test-renderer'
import { ListImgLoad, ListTitleLoad, ListDescriptionLoad, LoadWrapper, LoadingAnimation, StyledImage, ListContainer, Container } from './productDetails.styles'

test('it works', () => {
    const tree = renderer.create(<ListDescriptionLoad />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<ListImgLoad selectedDisplay={'list'} />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<ListTitleLoad />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<LoadWrapper />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<LoadingAnimation />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<StyledImage />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<ListContainer selectedDisplay={'list'} />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works', () => {
    const tree = renderer.create(<Container selectedDisplay={'list'} />).toJSON()
    expect(tree).toMatchSnapshot()
})