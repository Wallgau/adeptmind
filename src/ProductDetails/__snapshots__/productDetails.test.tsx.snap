// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<p
  className="sc-hKFxyN iNHhXC"
/>
`;

exports[`it works 2`] = `
<div
  className="sc-gtsrHT cqNcI"
/>
`;

exports[`it works 3`] = `
<h2
  className="sc-dlnjwi LVolq"
/>
`;

exports[`it works 4`] = `
<div
  className="sc-bdnxRM diaxRO"
/>
`;

exports[`it works 5`] = `
<div
  className="sc-eCApnc lbjCBy"
/>
`;

exports[`it works 6`] = `
<img
  className="sc-iCoGMd kGRhEU"
/>
`;

exports[`it works 7`] = `
<li
  className="sc-gKAaRy jlOQtp"
/>
`;

exports[`it works 8`] = `
<div
  className="sc-jSFjdj jmkOZz"
/>
`;
